bin/evaluate.sh

```diff
@@ -2,6 +2,13 @@
 
 set -o errexit
 
+backtrace="--backtrace"
+if [ "${*#*$backtrace}" != "$*" ]; then
+    export RUST_BACKTRACE=1
+    set -- ${*%$backtrace*}${*#*$backtrace}
```

Comment:
Could the caller of this script just set the environment variable for the subprocess? This seems like crazy bash syntax that only one person in the world will ever understand :(